models/topics.model.js

```diff
@@ -1,4 +1,5 @@
 const db = require('../db/connection')
+const fs = require('fs')
```

Comment:
We should use fs to read the endpoints.json file and return its content. The model should then be invoked in the controller